Misc/NEWS.d/next/Library/2018-06-17-21-02-25.bpo-32498.La3TZz.rst

```diff
@@ -0,0 +1,2 @@
+Made ``urllib.parse.unquote()`` polymorphic, accepting both string and
```

Comment:
"polymorphic" doesn't help clarity here; IMO just write "Made ... accept both str and bytes."